site/docs/plugins/nestjs.md

```diff
@@ -0,0 +1,27 @@
+---
+prev: false
+next: false
+---
+
+# NestJS Integration (`nestjs`)
+
+This plugin provides an integration of a grammY bot into a [NestJS](https://nestjs.com/) application.
+
+## Installation
+
+Run
+
+```sh
+npm install @grammjys/nestjs
+```
+
+to install the plugin in your existing NestJS project.
+
+## Usage
+
+todo
```

Comment:
@NarHakobyan please add a short example of a basic usage of this plugin by editing this file. It does not have to be anything fancy, just a simple echo bot to get people started maybe. I think it is fair to require familiarity with NestJS from our readers, so they should already have a working NestJS project when they get to this step.